pkg/client/didexchange/client_test.go

```diff
@@ -897,7 +897,7 @@ func TestClient_QueryConnectionsByParams(t *testing.T) {
 		const state = "completed"
 		for i := 0; i < count; i++ {
 			val, e := json.Marshal(&connection.Record{
-				ConnectionID: string(i),
+				ConnectionID: fmt.Sprint(i),
```

Comment:
is there any difference?

Reply:
I had an error in go 1.15. This is why I fixed it.

Reply:
looks like it closes it :)